frontend/package.json

```diff
@@ -38,7 +38,7 @@
     "package": "func() { yarn workspace @clutch-sh/\"$@\"; }; func",
     "publishBeta": "lerna run compile && lerna run publishBeta --no-bail",
     "start": "yarn clean && yarn compile:watch & FORCE_COLOR=true yarn workspace @clutch-sh/app start | cat",
-    "storybook": "start-storybook -p 6006",
+    "storybook": " rm -rf node_modules/.cache/storybook/ && start-storybook -p 6006 -h localhost",
```

Comment:
There is an issue in the latest storybook version where assets are not purged when running in dev mode.